RecoHI/HiJetAlgos/plugins/HiL1Subtractor.cc

```diff
@@ -7,9 +7,9 @@ using namespace std;
 //
 // constants, enums and typedefs
 //
-double puCent[11] = {-5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5};
-double medianPtkt[12];
-
+namespace {
+  double constexpr puCent[11] = {-5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5};
```

Comment:
From CLANG

```
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_12_6_X_2022-10-05-2300/src/RecoHI/HiJetAlgos/plugins/HiL1Subtractor.cc:11:20: warning: unused variable 'puCent' [-Wunused-const-variable]
```

Reply:
Thanks (I was traveling yesterday so couldn't check on the failure).